easybuild/easyconfigs/m/MultiQC/MultiQC-0.7-goolf-1.7.20-Python-2.7.11.eb

```diff
@@ -0,0 +1,78 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Bundle'
+
+name = 'MultiQC'
+version = '0.7'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://multiqc.info/'
+description = """Aggregate results from bioinformatics analyses across many samples into a single report"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+parallel = 1
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('matplotlib', '1.5.1', '-Python-%(pyver)s'),
+    # numpy dependency provided by python module
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('six', '1.10.0', {
+	'source_urls': ['https://pypi.python.org/packages/source/s/six/'],
+    }),
+    ('pyparsing', '2.1.9', {
+	'source_urls': ['https://pypi.python.org/packages/source/p/pyparsing/'],
+    }),
+    ('pytz', '2016.6.1', {
+	'source_urls': ['https://pypi.python.org/packages/source/p/pytz/'],
+    }),
+    ('cycler', '0.10.0', {
+	'source_urls': ['https://pypi.python.org/packages/source/c/cycler/'],
+    }),
+    ('python-dateutil', '2.5.3', {
+	'source_urls': ['https://pypi.python.org/packages/source/p/python-dateutil/'],
+	'modulename': 'dateutil',
+    }),
+    ('MarkupSafe', '0.23', {
+	'source_urls': ['https://pypi.python.org/packages/source/m/markupsafe'],
+    }),
+    ('click', '6.6', {
+	'source_urls': ['https://pypi.python.org/packages/source/c/click/'],
+    }),
+    ('PyYAML', '3.12', {
+	'source_urls': ['https://pypi.python.org/packages/source/p/pyyaml'],
+	'modulename': 'yaml',
+    }),
+    ('simplejson', '3.8.2', {
+	'source_urls': ['https://pypi.python.org/packages/source/s/simplejson/'],
+    }),
+    ('Jinja2', '2.8', {
+	'source_urls': ['https://pypi.python.org/packages/source/J/Jinja2/'],
+    }),
+    ('multiqc', version, {
+	#'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://pypi.python.org/packages/source/m/multiqc/'],
+    }),
+]
+
+modextrapaths = {
+    'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages'],
+    'LD_LIBRARY_PATH': ['lib/python%(pyshortver)s/site-packages']
+}
+
+sanity_check_paths = {
+    'files': ['bin/multiqc'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/multiqc-%(version)s-py%(pyshortver)s.egg']
```

Comment:
@pescobar since the `.egg` may be a file too, it's better to just check for `'lib/python%(pyshortver)s/site-packages`